### ISSUE TITLE
How do I compile multiple files?

### ISSUE THREAD

**Issue by atrauzzi** [MikeyAlder/laravel-elixir-typescript#1]
It doesn't look like I can pass an array to specify multiple root files?

**Comment by mikeygcooper** [comment]
I had it set up so that it processes all files that are in the resources/typescript/ folder and then concatenates them all into the single js filename that you pass through.

Would you prefer I change it so that it just spits out the files without merging into one file (or perhaps make that part an optional flag?)

**Comment by atrauzzi** [comment]
Yeah, definitely.  What it should receive is an array of files.  Then it compiles those files individually to a single `.js` for each file in the array, more or less like profiles.

If the files have dependencies, they can use imports rather than depending on manual combining.

**Comment by atrauzzi** [comment]
@MikeyAlder Is this something you're working on currently?  I'm looking forward to the change as I have multiple entrypoints in my project that need separate build runs. :)

**Comment by mikeygcooper** [comment]
I haven't had chance to look at this yet but will over the next couple of
days for you. Have been very busy with work at the moment. :) sorry for any
inconvenience :)

**Comment by atrauzzi** [comment]
That would be great, thanks so much! :)
